ursaspace/ursa-arcgis-toolbox
# feature/vcm-7209: Analytics Ordering

# VCM-7209: Analytics Ordering

## Whats Changed
- New `Analytic` Tool in Toolbox (w/ XML file for tool metadata)
- Moved custom arcpy parameters to `parameters` directory
- Moved all source code files into `src` directory (less confusing to non-technical folks who need to install this)
- Updated root README, moved technical details to separate readme in `src`

## Notes
- Ran prettier on all files and rearranged the project structure (so more changes than usual)
- Trevor wrote the README, so added some more details on the analytics ordering following his pattern
- Ordering works with all internal accounts and external L2 accounts that are setup for invoicing. L1 accounts or external accounts that are not setup for invoicing will fail to submit an order.
- The UI controls in the toolbox are pretty limited. Followed the same behaviors that were approved for the tasking tool. Not many other improvements we can make to the UX.
- You will need access to the ArcGIS Instance in AWS (reach out to IT for access, say its for the toolbox). You'll also need an ArcGIS login with a license to ArcGIS Pro (IT can set you up with that as well)

## To Test
- Run tests (see readme)
- Submit an analytic order! See confirmation email with order details

<img width="1728" alt="Screenshot 2023-05-01 at 10 38 08 AM" src="https://user-images.githubusercontent.com/123033437/235469084-7589e0f2-72ff-453e-8714-05fbdafd2b8c.png">
